docs/services/metadata/schema-metadata.json

```diff
@@ -304,6 +307,28 @@
       ],
       "additionalProperties": false
     },
+    "publication": {
+      "type": "object",
+      "description": "A publication.",
+      "properties": {
+        "text": {
```

Comment:
The url type already contains an optional `text` property....

The way you have defined the properties structure these are valid examples of a publication object:
```json
{
  "text" : "Only text is mandatory"
}
```

```json
{
  "text" : "Only text is mandatory",
  "url" : [
       {
          "url" : "https://example.com/1",
          "text" : "Url may contain a text, the publication url is a list"
       },
        {
          "url" : "https://example.com/@"
       }
  ]
}

Reply:
Is this intended like so?

Reply:
yes, this is intended:  
for URLs the text is what is supposed to be displayed within the `<a>` tag.  
for publications, we do not intend to make the entire publication text the `<a>` tag, only parts of it.  
This is conventional in citing in humanities.

Reply:
Random example I got from the internet:

Alonso, Alvaro, and Julio A. Camargo. "Toxicity of Nitrite to Three Species of Freshwater Invertebrates." Environmental Toxicology, vol. 21, no. 1, 3 Feb. 2006, pp. 90-94. Wiley Online Library, doi: [10.1002/tox.20155](https://doi.org/10.1002/tox.20155).

```
Alonso, Alvaro, and Julio A. Camargo. "Toxicity of Nitrite to Three Species of Freshwater Invertebrates." Environmental Toxicology, vol. 21, no. 1, 3 Feb. 2006, pp. 90-94. Wiley Online Library, doi: [10.1002/tox.20155](https://doi.org/10.1002/tox.20155).
```

This would roughly map to

```json
"publications": [
    {
        "text": "Alonso, Alvaro, and Julio A. Camargo. \"Toxicity of Nitrite to Three Species of Freshwater Invertebrates.\" Environmental Toxicology, vol. 21, no. 1, 3 Feb. 2006, pp. 90-94. Wiley Online Library, doi:"
        "url": [
            "text": "10.1002/tox.20155"
            "url": "https://doi.org/10.1002/tox.20155"
        ]
    }
]
```

Does that make sense now?